tests/test_roman.py

```diff
@@ -613,13 +612,13 @@ def test_roman_psfs():
     # - achromatic PSFs without loading the pupil plane image.
```

Comment:
Is this still true?  (We have some caveats about things we do/don't test, but since you fixed the defaults to be more accurate and faster, I think we do test some of what's mentioned here as not being tested?)